modules/ROOT/pages/netsuite/netsuite-examples.adoc

```diff
@@ -0,0 +1,560 @@
+    </netsuite:config>
+
+    <flow name="netsuite-search-employee">
+    		<http:listener doc:name="/employee" config-ref="HTTP_Listener_config"
+    			path="/employee" />
+    		<ee:transform doc:name="Search Criteria">
+    			<ee:message >
+    				<ee:set-payload ><![CDATA[%dw 2.0
+    output application/java
+    ---
+    {
+    	isInactive: {
+    		searchValue: attributes.queryParams["isInactive"]
+    	},
+    	lastName: {
+    		operator: "STARTS_WITH",
+    		searchValue: attributes.queryParams["lastName"]
+    	}
+    } as Object {
+    	class : "org.mule.module.netsuite.extension.api.EmployeeSearchBasic"
+    }]]></ee:set-payload>
+    			</ee:message>
+    		</ee:transform>
+    		<netsuite:search doc:name="Search Employee"config-ref="NetSuite_Config" key="EMPLOYEE_BASIC" pageSize="10" />
+    		<ee:transform doc:name="Response to Json">
+    			<ee:message >
+    				<ee:set-payload ><![CDATA[%dw 2.0
+    output application/json
+    ---
+    payload]]></ee:set-payload>
+    			</ee:message>
+    		</ee:transform>
+    	</flow>
+</mule>
+----
+
+[[customer-advanced-search]]
+== Customer Advanced Search Filtering by the Date and Array fields
+
+This example shows how to use the `Search` operation to obtain the customer records that were created after 2015 and that are in the "LEAD" stage.
+
+For this example to work, there must be at least one customer that meets the search conditions in the NetSuite instance.
+
+The following screenshot shows the Studio app flow for this example:
+
+.Customer advanced search app flow in Studio
+image::netsuite/netsuite-examples-search-customer.png[App flow for the Customer advanced search example]
+
+Creating this example involves creating a new Mule project and configuring HTTP Listener, the NetSuite Search operation, three Transform Message components, and a For Each
+
+=== Configure HTTP Listener
+
+Configure HTTP Listener to initiate a Mule flow when a call is made to the `/customer` path on localhost port 8081:
+
+. In Studio, click *HTTP* and drag the *Listener* operation to the canvas.
+. Change the display name of the *Listener* operation to `/customer`.
+. Select an existing global element or create a new one for the *Listener* operation.
+. On the Listener properties window, set the *path* field value to `/customer`.
+
+=== Add the NetSuite Search Operation
+
+. From the *Mule Palette* view, select *NetSuite* and drag the *Search* operation to the right of *Listener*.
+. Change the display name of the *Search* operation to `Search Customer Advanced`.
+. Select an existing global element or create a new one for the *Search* operation.
+. On the properties window:
+** In the *Key* field, select `CUSTOMER_ADVANCED`.
+** Set the *Page Size* field value to an integer value between `10` and `1000`.
+
+=== Add the First Transform Message Component
+
+This Transform Message Component configures custom labels for the search results and specifies the search criteria for the flow:
```

Comment:
Note the formatting difference between this topic intro and the following one (This *Transform Message* component converts the response to JSON format:)

Please use your best judgment about formatting these items correctly, based on the "font choices" table in our style reference, recently approved by the team.